cdc/api/v2/api.go

```diff
@@ -14,25 +14,62 @@
 package v2
 
 import (
+	"github.com/pingcap/tiflow/cdc/model"
+	"github.com/pingcap/tiflow/pkg/upstream"
+	"net/http"
+
 	"github.com/gin-gonic/gin"
 	"github.com/pingcap/tiflow/cdc/api/middleware"
 	"github.com/pingcap/tiflow/cdc/capture"
+	"github.com/pingcap/tiflow/cdc/owner"
 )
 
 // OpenAPIV2 provides CDC v2 APIs
-type OpenAPIV2 struct {
+type openAPIV2 struct {
 	capture *capture.Capture
+	// use for unit test only
+	testStatusProvider owner.StatusProvider
```

Comment:
I think adding a field solely for the purpose of testing is not good for code maintenance. Can you consider using a `StatusProvider` in production code path too? Maybe we can remove the dependence on `capture.Capture`.

Reply:
will remove the dependence on *capture later